### PR TITLE
Add search help button to home page that will lead to search help page

### LINKED ISSUE #172: Add button under seek help map on home page that will lead to search for help page

### PR DESCRIPTION
**Pull request by jennetturkkan**
* Added button
* Added Arabic translation
Screen Shot 2020-11-22 at 20 55 20

### CODE REVIEW

File: src/components/SeekHelpCarousel/index.jsx [hunk 1]
```diff
@@ -31,6 +32,11 @@ const SeekHelpCarousel = () => {
           </Slider>
         </Col>
       </Row>
+      <Link to="/seekhelp">
+        <h6 className="searchHelpBtn">
```
Comment (hiba-machfej) [inline src/components/SeekHelpCarousel/index.jsx]:
```
   <h6>
            {t("home.buttons. searchHelpBtn")}
            {document.body.dir === "rtl" ? (
              <span> &#8592; </span>
            ) : (
              <span> &#8594; </span>
            )}
          </h6>
```

Comment (jennetturkkan) [reply src/components/SeekHelpCarousel/index.jsx]: okay

Comment (hiba-machfej) [reply src/components/SeekHelpCarousel/index.jsx]: can you make the code like this so the arrow flips between Arabic and English and check if its work

**Review by hiba-machfej — APPROVED** [review]

File: src/components/SeekHelpCarousel/index.jsx (hunk 1 above)
Comment (jennetturkkan) [reply src/components/SeekHelpCarousel/index.jsx]: looks like this
Screen Shot 2020-11-22 at 21 08 24

Comment (jennetturkkan) [reply src/components/SeekHelpCarousel/index.jsx]: but should be like this I guess, right?
Screen Shot 2020-11-22 at 21 09 06

Comment (hiba-machfej) [reply src/components/SeekHelpCarousel/index.jsx]: try adding it in a row

Comment (jennetturkkan) [reply src/components/SeekHelpCarousel/index.jsx]: is second or first option right? i mean arrows

**Comment by jennetturkkan** [comment]
@louisrli i will work on this PR tomorrow, please don't approve it. can't believe myself that I am saying it :D

**Comment by jennetturkkan** [comment]
@louisrli actually I fixed it now. can you check please?

Screen Shot 2020-11-23 at 00 12 40

**Comment by louisrli** [comment]
no. you told me not to approve it

**Review by louisrli — APPROVED** [review]

**Comment by jennetturkkan** [comment]
@louisrli :D @Ammar-64 can you merge please?

**Pull request merged**